keyboards/anne_pro/anne_pro_lighting.c

```diff
@@ -192,3 +196,35 @@ void anne_pro_lighting_set_keys(uint8_t keys, uint8_t *payload) {
     uart_tx_ringbuf_write(&led_uart_ringbuf, 5, buf);
     uart_tx_ringbuf_write(&led_uart_ringbuf, 5 * keys, payload);
 }
+
+/* Set Caps Lock red led on */
+void anne_pro_lighting_caps_lock_on(){
+		if(!leds_enabled && !not_init){
+			/* Wake up the LED controller if off */
+			writePinHigh(C15);
+			chThdSleepMilliseconds(50);
+			not_init=true;
+		}
+		if(leds_enabled) // solves problems when LT(1,KC_CAPS) is used in keymap(Tap = caps, Hold = MO(1))
+			anne_pro_lighting_update();
+		uart_tx_ringbuf_write(&led_uart_ringbuf,4,"\x09\x0c\x0c\x01");
+		uart_tx_ringbuf_start_transmission(&led_uart_ringbuf);
+		prev_state_caps_lock = true;
+}
+/* Set Caps Lock red led off */
+void anne_pro_lighting_caps_lock_off(){
+		if(leds_enabled) // solves problems when LT(1,KC_CAPS) is used in keymap(Tap = caps, Hold = MO(1))
+			anne_pro_lighting_update();
```

Comment:
Again, why do you need this here?

Reply:
same as on https://github.com/msvisser/qmk_firmware/pull/5#discussion_r305360254

Reply:
not needed any more